crates/solver/src/metrics.rs

```diff
@@ -132,8 +132,7 @@ impl Metrics {
                 "Surplus ratio differences between winning and best settlement per order",
             )
             .buckets(vec![
-                -2.0, // Difference of percentages can not exceed +/- 2
-                -1.0, -0.1, -0.01, -0.005, 0., 0.005, 0.01, 0.1, 1.0, 2.0,
+                -2.0, -1.0, -0.1, -0.01, -0.005, 0., 0.005, 0.01, 0.1, 1.0, 2.0,
```

Comment:
Honestly, I don't see much value in separating `>200% surplus difference ; 200%-100% surplus difference` and `>100% surplus difference`.

Thanks for fixing the comment.

Reply:
I can also remove the +/- 2 (they were really only introduced when I thought they were the absolute bounds). will remove.